cocos/scripting/js-bindings/manual/physics3d/jsb_cocos2dx_physics3d_manual.cpp

```diff
@@ -265,7 +265,7 @@ bool jsb_cocos2d_Physics3DObject_setCollisionCallback(JSContext *cx, uint32_t ar
         JSB_PRECONDITION2( cobj, cx, false, "jsb_cocos2d_Physics3DObject_setCollisionCallback : Invalid Native Object");
 
         std::function<void(const Physics3DCollisionInfo &)> arg0;
-        std::shared_ptr<JSFunctionWrapper> func(new JSFunctionWrapper(cx, args.get(1).toObjectOrNull(), args.get(0)));
+        std::shared_ptr<JSFunctionWrapper> func(new JSFunctionWrapper(cx, JS::RootedObject(cx, args.get(1).toObjectOrNull()), args.get(0)));
```

Comment:
This two form have exactly the same result, the bottom one only create one more local variable.
To ensure object in JSFunctionWrapper won't get garbage collected, I'm using PersistentRootedObject in JSFunctionWrapper:

https://github.com/cocos2d/cocos2d-x/pull/14478/files#diff-3324b4ed095146d64860da926a8409ebR93